app/services/ingest_csv_row.rb

```diff
@@ -280,6 +281,14 @@ def aid_type_mapping
     end
   end
 
+  def oda_eligibility_mapping
+    @oda_eligibility_mapping ||= begin
+      yaml_to_objects(entity: "activity", type: "oda_eligibility")
```

Comment:
Just to check I understand: does this load data from the file `vendor/data/codelists/IATI/2_03/activity/oda_eligibility.yml` that was added in the previous commit?

Reply:
Yes, it takes the names and the codes from that file.